browser/brave_wallet/external_wallets_importer.cc

```diff
@@ -428,6 +428,7 @@ void ExternalWalletsImporter::GetMnemonic(bool is_legacy_crypto_wallets,
   if (!mnemonic) {
     VLOG(0) << "Failed to find mnemonic in decrypted keyrings";
     std::move(callback).Run(false, ImportInfo(), ImportError::kJsonError);
+    return;
```

Comment:
this is a bit of a behavioral change - but makes sense to me
cc: @darkdh

Reply:
This return was supposed to be here. If you look the next line you'll notice that not returning here will result in a crash.

Reply:
that is right, we need return here